### PR TITLE
Fix timeout in payment-is-showing.https.html

### PR DESCRIPTION
**Pull request by danyao**
The changed test was timing out before because of three reasons:

1. The onload event of the popup never fired after navigation.
2. The test assumed that `popupShowPromise` would reject when the popup navigates. This is not true in all implementations (i.e. Chrome) and this behavior has been removed from the spec. See https://github.com/w3c/payment-request/issues/872
3. This test file contains many test cases that use `test_driver.bless()`. When run from the web-based runner, the human tester may not be able to click through all steps within the default timeout of 1 minute.

(1) is fixed by adding an onload handler to the popup page, though I don't fully understand why.

(2) is fixed by replacing the `promise_rejects` expectation with a `step_timeout` to check that a new payment sheet can be opened successfully. This is sufficient to verify that the old payment sheet was indeed closed.

(3) is fixed by increasing the timeout by adding `<meta name="timeout" content="long">`.

### CODE REVIEW

File: payment-request/blank.html [hunk 1]
```diff
@@ -1 +1,14 @@
-<!DOCTYPE html> <meta charset="utf-8" />
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Blank Page</title>
+    <script>
+      window.onload = function(event) {
+        console.log("Loaded ", window.location.href);
```
Comment (marcoscaceres) [inline payment-request/blank.html]:
```suggestion
```

Comment (marcoscaceres) [reply payment-request/blank.html]: We can leave the no op, but no need for the console.log()

**Review by marcoscaceres — APPROVED** [review]
Looks good. Just a small suggestion.

File: payment-request/blank.html (hunk 1 above)
Comment (danyao) [reply payment-request/blank.html]: Done. I added a comment to document why this empty onload function is necessary.

**Review by rsolomakhin — APPROVED** [review]

**Pull request merged**